test/test_techcrunch.py

```diff
@@ -0,0 +1,39 @@
+from test.conftest import MockUQCSBot, TEST_CHANNEL_ID
+from unittest.mock import patch
+
+import feedparser
+from typing import List, Dict
+
+def get_local_techcrunch_data() -> List[Dict[str, str]]:
+    """
+    Returns the RSS data from local mock copy of techcrunch xml.
+    """
+    try:
+        rawdata = open("test/techcrunch.xml").read()
```

Comment:
Can we use `with open(...)` instead? This makes sure to clean up the opened file.